src/components/interactive-builder/add-question.modal.tsx

```diff
@@ -173,6 +178,7 @@ const AddQuestionModal: React.FC<AddQuestionModalProps> = ({
   const handleAnsConceptChange = (event: React.ChangeEvent<HTMLInputElement>) =>
     setConceptAnsToLookup(event.target.value);
 
+  const handleOrdersChange = (event: React.ChangeEvent<HTMLInputElement>) => setConceptToLookup(event.target.value);
```

Comment:
Nit: empty line between the two functions
```suggestion
  const handleOrdersChange = (event: React.ChangeEvent<HTMLInputElement>) => setConceptToLookup(event.target.value);
  
```